modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Settings.tsx

```diff
@@ -0,0 +1,274 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import {Option, Picker, Text} from '@clayui/core';
+import DropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import {fetch, navigate} from 'frontend-js-web';
+import React, {useEffect, useState} from 'react';
+
+import {IFDSViewSectionProps} from '../FDSView';
+import {
+	API_URL,
+	DEFAULT_VISUALIZATION_MODES,
+	OBJECT_RELATIONSHIP,
+} from '../utils/constants';
+import openDefaultFailureToast from '../utils/openDefaultFailureToast';
+import openDefaultSuccessToast from '../utils/openDefaultSuccessToast';
+import {TVisualizationMode} from '../utils/types';
+
+const NOT_CONFIGURED_VISUALIZATION_MODE = {
+	label: Liferay.Language.get('go-to-visualization-modes'),
+	thumbnail: 'plus',
+	type: Liferay.Language.get('not-configured'),
+};
+
+const Settings = ({
+	fdsView,
+	fdsViewsURL,
+	onActiveSectionChange,
+	onFDSViewUpdate,
+	spritemap,
+}: IFDSViewSectionProps) => {
+	const [defaultVisualizationMode, setDefaultVisualizationMode] = useState(
+		NOT_CONFIGURED_VISUALIZATION_MODE.type
+	);
+	const [visualizationModes, setVisualizationModes] = useState<
+		Array<TVisualizationMode>
+	>([]);
+
+	const getActiveVisualizationModes = async () => {
+		const fields = [
+			OBJECT_RELATIONSHIP.FDS_VIEW_FDS_CARDS_SECTION,
+			OBJECT_RELATIONSHIP.FDS_VIEW_FDS_LIST_SECTION,
+			OBJECT_RELATIONSHIP.FDS_VIEW_FDS_FIELD,
+		].join(',');
+
+		const response = await fetch(
```

Comment:
As @dsanz said, this endpoint works like a charm. We should consider this request to a higher component (FDSView) and pass the data to other components. This way we could eliminate also the requests needed for the different Visualization Modes.